data-plane/core/controller/proto/v1/controller.proto

```diff
@@ -72,10 +72,7 @@ enum ConnectionType {
 }
 
 message ConnectionEntry {
-  uint64 id = 1;
-  ConnectionType connection_type = 2;
-  string ip = 3;
-  uint32 port = 4;
+  string config_data = 1;
```

Comment:
In the grpc ClientConfig we don't have id, connection_type. So should we keep this in our config_data ?